app/src/docs/components/DocsLayout/Navigation/index.jsx

```diff
@@ -122,8 +145,14 @@ class Navigation extends React.Component<Props, State> {
                     container: responsive,
                 })}
                 >
-                    {!!responsive && this.parseCurrentPage()}
-                    {this.parseNavigation()}
+                    {responsive && (
```

Comment:
`!!responsive`, if `responsive` is ever `0` or `NaN` then it'll render those as text. Low cost, guarantees no edge case surprises.